geoportal/geoportailv3_geoportal/static-ngeo/js/print/PrintController.js

```diff
@@ -270,9 +277,15 @@ const exports = function($scope, $window, $timeout, $q, gettextCatalog,
       console.assert(postcomposeListenerKey === null);
       postcomposeListenerKey = listen(this.map_,
           olRenderEventType.POSTCOMPOSE, postcomposeListener);
+      const bgLayer = this.backgroundLayerMgr_.get(this.map_);
+      const mbMap =  bgLayer.getMapBoxMap();
+      const data = JSON.stringify(mbMap.getStyle());
+      this.appMvtStylingService_.publishStyle(bgLayer, data);
```

Comment:
Shouldn't the serial be checked like when switching between 2D and 3D?
That code could be shared in a single place (a service).